package/base-files/files/bin/config_generate

```diff
@@ -160,18 +159,20 @@ generate_network() {
 
 	case "$protocol" in
 		static)
-			local ipad
+			local ipad netm bitm
 			case "$1" in
 				lan) ipad=${ipaddr:-"192.168.1.1"} ;;
 				*) ipad=${ipaddr:-"192.168.$((addr_offset++)).1"} ;;
 			esac
 
 			netm=${netmask:-"255.255.255.0"}
+			bitm=$(awk -v "netm=$netm" -e 'BEGIN { split(netm, octets, ".")
```

Comment:
Maybe this could become easier with #13765.
@pprindeville what do you think?

Reply:
Updated the code using the linked PR as a dependency.

Reply:
It shouldn't just depend on the PR, it should leverage it is @yogo1212's point.

We're trying to deprecate the use of awk.